external_search.py

```diff
@@ -1297,7 +1490,7 @@ def from_worklist(cls, _db, worklist, facets):
     def __init__(self, collections=None, media=None, languages=None,
                  fiction=None, audiences=None, target_age=None,
                  genre_restriction_sets=None, customlist_restriction_sets=None,
-                 facets=None
+                 facets=None, order=None, order_ascending=False
```

Comment:
I think it might be better not to have these arguments -- it could be redundant with what's in `facets`.